iree/vm/ops.h

```diff
@@ -17,6 +17,8 @@
 
 #include <stdint.h>
 
+#include "iree/vm/api.h"
```

Comment:
this should be iree/base/api.h (otherwise there's an implicit cycle with the VM depending on itself - vm/api.h is "all of the VM stuff for external users to depend on")